app/classifier/index.cjsx

```diff
@@ -40,6 +41,7 @@ Classifier = React.createClass
     subject: null
     classification: null
     onLoad: Function.prototype
+    cacheClassification: new CacheClassification
```

Comment:
Should this move into the `CachedClassification` component, so that there's only ever one instance of `CachedClassification` in the interface?